main.nf

```diff
@@ -38,7 +38,7 @@ def helpMessage() {
       -profile                      [str] Configuration profile to use. Can use multiple (comma separated)
                                           Available: conda, docker, singularity, test, awsbatch, <institute> and more
       --step                       [list] Specify starting step (only one)
-                                          Available: mapping, prepare_recalibration, recalibrate, variant_calling, annotate, Control-FREEC
+                                          Available: mapping, prepare_recalibration, recalibrate, variant_calling, annotate, ControlFREEC
```

Comment:
```suggestion
                                          Available: mapping, prepare_recalibration, recalibrate, variant_calling, annotate, Control-FREEC
```
Actually as for tools, the string is stripped out of `_` and `-` and lowercased, so `Control-FREEC` should actually work.
I would prefer that spelling as it's the one from the tool itself

Reply:
Agreed!